src/game/game/server.rs

```diff
@@ -336,9 +335,13 @@ impl GameServer {
 }
 
 #[derive(actix::Message, Serialize, Clone)]
-#[rtype(result="Arc<(actix::Addr<ClientSession>, bool)>")]
+#[rtype(result="Arc<(Option<actix::Addr<ClientSession>>, bool)>")]
```

Comment:
Faudrait tester si on a toujours besoin du `Arc`. Ça nous enlèverait beaucoup d'overhead pour un simple message.

Reply:
Faut creuser du côté du type [MessageResult in actix::prelude - Rust](https://docs.rs/actix/0.12.0/actix/prelude/struct.MessageResult.html)